tests/dist/chats.spec.dev.js

```diff
@@ -180,37 +187,38 @@ describe('chat resolvers', function () {
   test('remove direct chat', function _callee4() {
     var removeDirectChatResponse, removeDirectChatData;
     return regeneratorRuntime.async(function _callee4$(_context4) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context4.prev = _context4.next)) {
           case 0:
             _context4.next = 2;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n    mutation{\n      removeDirectChat(chatId:"'
-                  .concat(createdDirectChatId, '", organizationId:"')
-                  .concat(
-                    createdOrgId,
-                    '") {\n        _id\n    }\n    }\n            '
-                  ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n    mutation{\n      removeDirectChat(chatId:"'
+                    .concat(createdDirectChatId, '", organizationId:"')
+                    .concat(
+                      createdOrgId,
+                      '") {\n        _id\n    }\n    }\n            '
+                    ),
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.